about.css

```diff
@@ -1,4 +1,4 @@
-@import url("https://fonts.googleapis.com/css2?family=Raleway&swap=display");
+@import url("https://fonts.googleapis.com/css2?family=Raleway&display=swap");
```

Comment:
Correct query parameter is display=swap
https://css-tricks.com/google-fonts-and-font-display/